hera_cal/apply_cal.py

```diff
@@ -246,11 +246,28 @@ def apply_cal(data_infilename, data_outfilename, new_calibration, old_calibratio
     # load old calibration solution
     if old_calibration is not None:
         old_hc = io.HERACal(old_calibration)
-        old_gains, old_flags, _, _ = old_hc.read()
+        old_hc.read()
+        # determine frequencies to load in old_hc that are close to hc
+        freqs_to_load = []
+        for f in old_hc.freqs:
+            atol = np.mean(np.diff(hc.freqs)) / 10.
```

Comment:
It's not good to have magic numbers like this in the code. This tolerance factor, 10, should be a kwarg, though I think 10 is a reasonable default.

Reply:
i've added this to the kwargs and set default to 10.